docs/gettingstarted.rst

```diff
@@ -15,44 +15,50 @@ ephemerides for these synthetic bodiess using OIF (Objects in Field), and show y
 .. note::
   All input data files in this example are white-space separated format solely for the ease of reading.   
 
-Creating Object Files
--------------------------
+Creating the Orbit and Physical Parameter Files For the Synthetic Small Bodies
+----------------------------------------------------------------------------------------
 The first step in the process is to generate a set of files which describe the orbital and physical parameters
-of the objects that we wish to study. Here we will generate a file called 'testorb.des', which contains
+of the objects that we wish to study. Here we will generate a file called 'sspp_testset_orbits.des', which contains
 the orbits of five objects
 
-.. literalinclude:: ../demo/testorb.des
+.. literalinclude:: ../demo/sspp_testset_orbits.des
```

Comment:
Is it worth renaming this file?

Reply:
yeah eventually. This might not be the final test set we use, so I figure it's worth leaving things at the moment, and I will replace when we've agreed what the final example set should look like